### DISCUSSION TITLE
Next Code Break - Blogging with Eleventy

### DISCUSSION THREAD

**Discussion by giscus** [cfjedimaster/raymondcamden2023#41] [Comments]
# Next Code Break - Blogging with Eleventy

My next Code Break will be Thursday, November 7

https://www.raymondcamden.com/2024/11/04/next-code-break-blogging-with-eleventy

**Comment by bobmonsour** [comment]
Bummed that I'll be missing this as we'll be out of the country, but will certainly be watching the recording.
